src/sentry/api/helpers/group_index/index.py

```diff
@@ -80,8 +80,17 @@ def build_query_params_from_request(
             query_kwargs["cursor"] = Cursor.from_string(request.GET.get("cursor"))
         except ValueError:
             raise ParseError(detail="Invalid cursor parameter.")
+
     has_query = request.GET.get("query")
-    query = request.GET.get("query", "is:unresolved").strip()
+    query = request.GET.get("query", None)
+    if query is None:
+        if features.has("organizations:issue-priority-ui", organization):
+            query = "is:unresolved issue.priority:[high, medium]"
+        else:
+            query = "is:unresolved"
```

Comment:
nit: it could be good for code organization to [move](https://refactoring.com/catalog/extractFunction.html) this to a helper method, like `_get_default_query`, but your call on that